From 86e95b2a522a5b517800ac32862b40676ccc658e Mon Sep 17 00:00:00 2001
From: jinghua <xigu@marvell.com>
Date: Thu, 23 Mar 2017 19:28:25 +0800
Subject: [PATCH 0696/1051] arm64: dts: marvell: armada-37xx: add watchdog node

This patch adds watchdog node describing controller
found in Armada 37xx SoCs, which is enabled by default.

Change-Id: Id447ca5a68cdc9d0b3aa8896f4ff44128bec61a9
Signed-off-by: jinghua <xigu@marvell.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-37xx.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
index 22ce7f107019..a23a3d930d35 100644
--- a/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-37xx.dtsi
@@ -86,6 +86,12 @@
 			ranges = <0x0 0x0 0xd0000000 0x2000000>;
 			dma-coherent;
 
+			wdt: watchdog-timer@8300 {
+				compatible = "marvell,armada-3700-wdt";
+				reg = <0xd064 0x4>, <0x8300 0x40>;
+				clocks = <&xtalclk>;
+			};
+
 			spi0: spi@10600 {
 				compatible = "marvell,armada-3700-spi";
 				#address-cells = <1>;
-- 
2.17.1

